client/landing/gutenboarding/onboarding-block/designs/index.tsx

```diff
@@ -1,24 +1,78 @@
-import DesignPicker, { getAvailableDesigns } from '@automattic/design-picker';
-import { useLocale } from '@automattic/i18n-utils';
+import { isEnabled } from '@automattic/calypso-config';
+import DesignPicker, {
+	FeaturedPicksButtons,
+	getAvailableDesigns,
+	useCategorization,
+} from '@automattic/design-picker';
+import { useLocale, englishLocales } from '@automattic/i18n-utils';
 import { Title, SubTitle, ActionButtons, BackButton } from '@automattic/onboarding';
 import { useDispatch, useSelect } from '@wordpress/data';
 import { useI18n } from '@wordpress/react-i18n';
-import { useEffect } from 'react';
+import classnames from 'classnames';
+import { useEffect, useMemo } from 'react';
 import * as React from 'react';
 import JetpackLogo from 'calypso/components/jetpack-logo'; // @TODO: extract to @automattic package
 import Badge from '../../components/badge';
 import useStepNavigation from '../../hooks/use-step-navigation';
 import { useTrackStep } from '../../hooks/use-track-step';
 import { useIsAnchorFm } from '../../path';
 import { STORE_KEY as ONBOARD_STORE } from '../../stores/onboard';
-import type { Design } from '@automattic/design-picker';
+import type { Design, Category } from '@automattic/design-picker';
 
 import './style.scss';
 
+// Ensures Blog category appears at the top of the design category list
+// (directly below the All Themes category).
+const sortBlogToTop = ( a: Category, b: Category ) => {
+	if ( a.slug === b.slug ) {
+		return 0;
+	} else if ( a.slug === 'blog' ) {
+		return -1;
+	} else if ( b.slug === 'blog' ) {
+		return 1;
+	}
+	return 0;
+};
+
+const Header: React.FunctionComponent = () => {
+	const { __ } = useI18n();
+	const locale = useLocale();
+	const isDesignPickerCategoriesEnabled = isEnabled( 'signup/design-picker-categories' );
+	const isAnchorFmSignup = useIsAnchorFm();
+	const { goBack } = useStepNavigation();
+	const title = isDesignPickerCategoriesEnabled ? __( 'Themes' ) : __( 'Choose a design' );
```

Comment:
The options shown in Gutenboarding are slightly different from Themes. I think we want to change Gutenboarding so that it's also grabbing real themes from the API instead of using designs, but in the mean time maybe it's more accurate to keep calling these "designs"?

But maybe the distinction isn't that important because maybe the user doesn't care about the distinction this early in the onboarding.

Reply:
Yep, I think we can just call it “Themes” and make it consistent with Hero Flow.